inc/captcha_fonts/read_me.html

```diff
@@ -59,33 +59,32 @@ <h3>
       </li>
     </ul>
     <ul>
-      <li>Q: How can I use this font in <a href="http://www.aol.com/aim/">AOL Instant
-      Messenger</a>, <a href="http://messenger.msn.com/">MSN Messenger</a>, <a href=
```

Comment:
Isn't this a third-party library file? Do we modify these?

Reply:
No clue ...
The quote marks were appearing broken though, so I was bound to touch that file.